tuf/api/pylintrc

```diff
@@ -1,5 +1,5 @@
 [MESSAGE_CONTROL]
-disable=fixme
+disable=fixme,E1101
```

Comment:
E1101 is `no-member`, a variable has been accessed which the type checker doesn't think is a valid member of that type. That makes sense, we're upcasting to the base 


Rather than disabling this globally, it would probably make more sense to disable it for only the `tuf.api.metadata` module (by including the `#pylint: disable=` comment at the top of the module's file. It may be cleaner to do this for the 4 specific fields by using the typechecker's `generated-members` configuration, although these aren't really generated members so perhaps not.